cmd/loop/instantout.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/lightninglabs/loop/instantout/reservation"
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/urfave/cli"
+)
+
+var instantOutCommand = cli.Command{
+	Name:  "instantout",
+	Usage: "perform an instant off-chain to on-chain swap (looping out)",
+	Description: `
+	Attempts to instantly loop out into the backing lnd's wallet. The amount
+	will be chosen via the cli.
+	`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
```

Comment:
we could use a `StringSliceFlag` here to avoid manually parsing the channels, see https://github.com/lightningnetwork/lnd/blob/master/cmd/lncli/cmd_open_channel.go#L126 for reference.